### ISSUE TITLE
"Failed to make link: Timed out" when trying to plumb a matrix room to a libera.chat irc channel

### ISSUE THREAD

**Issue by olasd** [matrix-org/matrix-appservice-irc#1497]
**Describe the bug**
When trying to bridge #swh:matrix.org to irc://irc.libera.chat/#swh, through the integration available on app.element.io, the bridging fails with a "Failed to make link: Timed out" message.
I am admin of the matrix room, which has been freshly created; I am op on the IRC channel.

When typing the channel name, the `matrixbridge1` bot joins the channel, and leaves it immediately (to populate the list of channel operators, I assume):

```
10:51:36    --> ╡ matrixbridge1 (~matrixbri@2001:470:69fc:105::1:3807) has joined #swh
10:51:37    <-- ╡ matrixbridge1 (~matrixbri@2001:470:69fc:105::1:3807) has left #swh
```

After submitting the form, I get a spinning wheel for a few seconds, then the error message screenshotted below.

The `@appservice:libera.chat` user gets invited to the matrix room, but never joins. I've tried repeating the process with and without revoking its invitation, to no avail.

**To Reproduce**
1. Open the IRC bridge integration process on app.element.io
2. Select liberachat as IRC network
3. Type channel information and operator name
4. Submit
5. See error

**Expected behavior**
I expected the irc bot to request confirmation for bridging on the libera.chat side, and then the bridging process to complete.

**Screenshots**
![Failed to make link: Timed out](https://user-images.githubusercontent.com/12877/143219077-fa41f304-e456-4eac-9605-a580f30a8399.png)

**Desktop:**
 - OS: Debian bookworm/sid amd64
 - Browser Firefox
 - Version 94.0

**Additional context**
I've checked that my IRC user has no caller-id or unregistered user blocking going on. a fresh connected user can PM me just fine.

**Comment by morguldir** [comment]
This looks like a duplicate of vector-im/element-web#19845, which should be fixed by matrix-org/matrix-react-sdk#7179, so you can try it out on develop.element.io and see if it works

The error message seems quite lacking in firefox, the error was a bit more detailed in chromium

**Comment by olasd** [comment]
> This looks like a duplicate of [vector-im/element-web#19845](https://github.com/vector-im/element-web/issues/19845), which should be fixed by [matrix-org/matrix-react-sdk#7179](https://github.com/matrix-org/matrix-react-sdk/pull/7179), so you can try it out on develop.element.io and see if it works
> 
> The error message seems quite lacking in firefox, the error was a bit more detailed in chromium

Thanks for following up (and sorry for dropping the ball on my end). The plumbing request did work in develop.element.io. I'll close this issue.
